src/display/TopBar/TopBar.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { describe, expect, it } from "vitest";
+import TopBar from "./TopBar";
+
+describe("<TopBar /> component", () => {
+  it("should render without errors", () => {
+    render(<TopBar />);
+    expect(screen.getByRole("toolbar")).toBeInTheDocument();
+  });
+
+  it("should render 3 children", () => {
+    render(
+      <TopBar
+        leftSection={<div>LEFT</div>}
+        centerSection={<div>CENTER</div>}
+        rightSection={<div>RIGHT</div>}
+      />,
+    );
+    expect(screen.getByText("LEFT")).toBeInTheDocument();
+    expect(screen.getByText("CENTER")).toBeInTheDocument();
+    expect(screen.getByText("RIGHT")).toBeInTheDocument();
+  });
+
+  it("should render 1 child", () => {
+    render(<TopBar leftSection={<div>LEFT</div>} />);
+    expect(screen.getByText(/LEFT/)).toBeInTheDocument();
```

Comment:
for the "getByText(/LEFT/)"
you have the choice, either you write it  : getByText("LEFT")
or you write it with a final "i" : getByText(/LEFT/i)
But I think in the getByText function with double quote is correct
Check every test that use it

Reply:
fixed